x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -122,15 +137,57 @@ const EventDetailsComponent: React.FC<Props> = ({
     [alerts, browserFields, data, id, isAlert, timelineId, timelineTabType]
   );
 
-  const selectedTab = useMemo(() => tabs.find((t) => t.id === view) ?? tabs[0], [tabs, view]);
+  const selectedEventTab = useMemo(() => tabs.find((t) => t.id === eventView) ?? tabs[0], [
+    tabs,
+    eventView,
+  ]);
+
+  const isThreatPresent: boolean = useMemo(
+    () =>
+      selectedEventTab.id === tabs[0].id &&
+      isAlert &&
+      !!data.find((item) => item.field === 'threat.indicator'),
```

Comment:
Does this logic imply that any alert with a `threat.indicator` object is going to count as a CTI alert? Are there any situations where we _wouldn't_ want that behavior?

If e.g. one wrote a query rule against _existing_ CTI alerts, I believe we would continue to display these details for the "secondary" CTI alerts that were generated from the original ones, even though the secondary alerts were not explicitly generated from an Indicator Match rule. This is _probably_ the correct behavior, but it's worth testing and verifying with product.

Reply:
this is an interesting one. I am not sure what the answer should be, who do you think would be a good contact to ask this question?

Reply:
My opinion as I'm writing the row renderer: I think logic should be based purely on the presence/absence of threat match fields on the alert, regardless of how they got there. By that logic, both Indicator Match alerts and meta-IM alerts would have these fields displayed, since they both have `threat.indicator` fields.

So: I agree that this logic is correct, but let's be sure to verify in our next product sync.